frank-doc-doclet/src/test/resources/doc/examplesExpected/category.json

```diff
@@ -31,14 +31,14 @@
             "elementNames": [
                 {
                     "labels": {
+                        "CategoryA": [
```

Comment:
Why did you change this order? It seems to break the test.

Reply:
When running locally, the test would fail because of the previous order. When looking at it together with Niels, we decided that the expected order could be updated in the result file.
But now that the test fails in CI, I think the test needs to be looked at so that the element order does not matter in the JSON compare.

Reply:
This is now fixed due to #357